project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -235,3 +229,36 @@ function c4m_user_notifications_notify_inactive_users() {
  */
 function c4m_user_notifications_notify_comment_on_content($comment) {
 }
+
+/**
+ * Email notification to group requester, that group is waiting for review.
+ */
+function c4m_user_notifications_notify_new_group_pending($node) {
+
+}
+
+/**
+ * Helper function.
+ *
+ * Returns ids of all users that have flagged $node, with $flag_name.
+ *
+ * @param object $node
+ *   Node object.
+ * @param string $flag_name
+ *   Name of the flag with which user flagged the node.
+ *
+ * @return array $uids
+ *   Array of user ids that flagged the node.
+ */
+function _c4m_user_notifications_get_flagging_uids($node, $flag_name) {
```

Comment:
@ordavidil 
This function formats the UIDs array, as required for message_subscribe_send _message().
I've simplified it to use flag_get_entity_flags() inside.